Penumbra/Mods/ModCollection.cs

```diff
@@ -156,6 +156,9 @@ public ModInfo AddModSettings( ResourceMod mod )
             var entry = new ModInfo
             {
                 Priority = ModSettings.Count,
+                CurrentGroup = 0,
+                CurrentTop = 0,

```

Comment:
you don't actually need to initialise these to 0 because you already do it for each in the class itself